components/community/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Flex, Image, VStack } from "@chakra-ui/react";
+import type { FC } from "react";
+import { Heading, Link, Text } from "tw-components";
+
+interface CommunityCardProps {
+  image: string;
+  link: string;
+  title: string;
+  description: string;
+}
+
+export const CommunityCard: FC<CommunityCardProps> = ({
+  image,
+  link,
+  title,
+  description,
+}) => {
+  return (
+    <Link href={link} _hover={{ textDecor: "none" }}>
```

Comment:
can't use a Link like this, use LinkBox with LinkOverlay, check how we do it in other places